### DISCUSSION TITLE
Obvious Function I Can't Find: How to add new Bookmarks?

### DISCUSSION THREAD

**Discussion by decard123** [mbnuqw/sidebery#730] [Q&A]
it should be obvious (i thought), but i cannot see how to add Bookmarks

**Comment by Qjo1** [comment]
If you are using the bookmark panel you can right click on any folder and create a new bookmark or folder.

In tab panels, right click on a tab and click the star symbol in the top right of the menu that opens.

![grafik](https://user-images.githubusercontent.com/107263952/186665333-793ac7f8-9791-4a6e-a81e-07c308d2aabc.png)

**Comment by decard123** [comment]
so to clarify, how do i save the current tab as bookmark?

**Reply by Qjo1** [reply]
In the sidebar, right click on that tab to open the menu as shown.

Or click on that star icon on the right side of Firefox's address bar. Both works. I don't know if the first option is available in v4 though.

**Reply by decard123** [reply]
thanks so much :-)

**Reply by decard123** [reply]
ah. the star at very top left when you RMC. i was looking for  asimple menu option
